gruntfile.js

```diff
@@ -179,7 +179,7 @@ module.exports = function(grunt) {
           {
             context: '/test/components',
             host: 'localhost',
-            port: '9001',
+            port: '9003',
```

Comment:
why do we change port?

Reply:
Because we can't run two web servers on same port.